README.md

```diff
@@ -61,6 +61,28 @@ s = shippo.Shippo(client=http_client)
 ```
 <!-- End Custom HTTP Client [http-client] -->
 
+## Debug HTTP Client
+
+The Python SDK returns schema models directly rather than wrapping the response in an envelope along with 
```

Comment:
I know this is pedantic but could we say "The Shippo Python SDK"?

Reply:
done!